documentation/design/console/global-console.adoc

```diff
@@ -0,0 +1,134 @@
+:toc:
+:toclevels: 4
+
+== Overview
+
+The Console is EnMasse comprises two separate web-applications that are hyperlinked together.
+
+* Global Console - that allows the viewing and management of address spaces.
+* Address Space Console - that allows the viewing and management of:
+ * addresses within an address space, and
+ * messaging connections made by applications within that address space.
+ * statistics
+
+In the long term, the desire is that current responsibilities of the Address Space Console will be migrated into the Global Console and the separate Address Space Consoles will be retired completed.
```

Comment:
Why wouldn't we plug in the Address Space Console, the same as we would for Strimzi?

Reply:
I'd always imagined the address space console being subsumed into the global console.  I don't see the point of maintain a separate component.  I agree that for other components, a link to another console might be the way.  I think there will be time for discussion when we look at the next phase.